pyensembl/__init__.py

```diff
@@ -1,5 +1,5 @@
-from ensembl_release import EnsemblRelease
-from gtf import GTF
-from locus import Locus
-from compute_cache import clear_cached_objects
-from reference_transcripts import ReferenceTranscripts
+from .ensembl_release import EnsemblRelease
```

Comment:
I had to look around for why they disable implicit relative imports. I guess the logic holds up but it's still an annoying change:

http://programmers.stackexchange.com/questions/159503/whats-wrong-with-relative-imports-in-python